docs/releases.mdx

```diff
@@ -12,6 +12,7 @@ Currently supported versions of Sourcegraph:
 
 | **Release**  | **General Availability Date** | **Supported** | **Release Notes**                                                  | **Install**                                          |
 |--------------|-------------------------------|---------------|--------------------------------------------------------------------|------------------------------------------------------|
+| 5.10 Patch 3 | December 2024                 | ✅             | [Notes](https://sourcegraph.com/docs/technical-changelog#v5103940) | [Install](https://sourcegraph.com/docs/admin/deploy) |
```

Comment:
@Chickensoupwithrice we need to make sure this is added as well. Right now it's by hand via a step in the release docs. 
This is the source of truth for releases though, so we need to be more diligent about updating this.